src/topic_based_system.cpp

```diff
@@ -38,11 +38,13 @@
 #include <rclcpp/executors.hpp>
 #include <topic_based_ros2_control/topic_based_system.hpp>
 
-static constexpr auto EPSILON = 1e-5;
-
 namespace topic_based_ros2_control
 {
 
+static constexpr std::size_t POSITION_INTERFACE_INDEX = 0;
+static constexpr std::size_t VELOCITY_INTERFACE_INDEX = 1;
+static constexpr std::size_t EFFORT_INTERFACE_INDEX = 3;
```

Comment:
Why did the index need to skip `2`?

Reply:
Yes, it's for acceleration, I'll add a note, one sec